config/locales/en/summary.yml

```diff
@@ -17,8 +18,22 @@ en:
       next_court_hearing: Next court hearing
       justification_for_legal_aid: Justification for legal aid
       passport_justification_for_legal_aid: Justification for legal aid
+      legal_representative_details: Legal representative details
```

Comment:
Tiny thing, this will be "Legal representative" as per design, if I'm correct. We can rename the summary class as well to be consistent as we don't use `details` suffix in any of the other section classes.